### DISCUSSION TITLE
Experiment:  Delegate acceptance test specification to Gherkin

### DISCUSSION THREAD

**Discussion by jondavid-black** [DevOps-MBSE/AaC#703] [Ideas]
Hypothesis:
We believe that allowing the definition of test specification in 3rd party Gherkin (i.e. Cucumber) for developers will simplify the system modelling and delivery flow.
We will know we are successful when we can still validate our models with data defined in 3rd party Gherkin and we see system modelers choose to use 3rd party Gherkin to define test specifications.

Note:  The current implementation of the Scenario may be too trivial to successfully perform this experiment.  First we need to reason about the value of validating content within the Scenario to other content in the model (i.e. inputs and outputs are correct).  Then we need to consider test case specification (i.e. Examples in Gherkin).  Although it shouldn't hold up this issue, we should look at other automated acceptance testing solutions and see what multiple delegation may look like.

**Comment by lizzcondrey** [comment]
This is currently being further evaluated and experimentally implemented within the AaC 4.0 work going on. So more thorough review and consideration will be able to be conducted once that experiment implementation is in place to determine any remaining holes.
